Assets/Scripts/Interactables/ItemPickup.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Managers;
+
+namespace Inventory
+{
+    public class ItemPickup : Interactable
+    {
+        [SerializeField] private Item item; //reference for items
+        [SerializeField] private InventoryController inventory;
+
+        private void Start()
+        {
+            GetInventory();
+        }
+
+        private InventoryController GetInventory()
```

Comment:
I think this isn't really needed, since you can simply drag in the reference to the inventory into the serialised field :)

Reply:
Oh okay. I thought we don't want to manually assign anything so...